starter_code/views/beers.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="beerDivs">
+  {{#each beers}}
+      {{> beerscard this}}
```

Comment:
Pas forcément besoin de mettre this ici, comme tu es dans une boucle, ton partial va trouver tout seul ce que this est : 
```
{{#each beers}}
      {{> beerscard this}}
{{/each}}
```